lib/pusher-fake/server.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require_relative "extensions/parent_trap"
```

Comment:
I'm not a library developer, so I'm not sure the best way to load and apply the module. Also considered support/base or support/rspec, or lib/pusher-fake where libraries are being `require`d